src/tools/miri/src/diagnostics.rs

```diff
@@ -362,7 +362,10 @@ fn report_msg<'tcx>(
         if is_local && idx > 0 {
             err.span_note(frame_info.span, &frame_info.to_string());
         } else {
-            err.note(&frame_info.to_string());
+            let sm = sess.source_map();
+            let lo = sm.lookup_char_pos(frame_info.span.lo());
+            let filename = sm.filename_for_diagnostics(&lo.file.name);
+            err.note(format!("{frame_info} at {}:{}:{}", filename, lo.line, lo.col.0 + 1));
```

Comment:
Isn't there a utility function somewhere that renders a span as a string? Doesn't feel great to have that logic here, in particular with that `+1`.

Reply:
Now that we only print this when we don't have a span to show, maybe using `span_to_string` and showing the end makes sense?

Reply:
no diff in CI because we filter too heavily in the miri test suite